spark/src/test/java/io/ddf/spark/content/SchemaHandlerTest.java

```diff
@@ -44,17 +44,47 @@ public void testApplySchema() throws DDFException {
 
     SparkDDF ddf = (SparkDDF) manager.sql2ddf("select * from DataIngestion", "SparkSQL");
     SchemaHandler schemaHandler = (SchemaHandler) ddf.getSchemaHandler();
-    List<Schema.Column> columns = new ArrayList<>();
-    columns.add(new Schema.Column("type_boolean", Schema.ColumnType.BOOLEAN));
-    columns.add(new Schema.Column("type_short", Schema.ColumnType.SMALLINT));
-    columns.add(new Schema.Column("type_long", Schema.ColumnType.BIGINT));
-    columns.add(new Schema.Column("type_double", Schema.ColumnType.DOUBLE));
-    columns.add(new Schema.Column("type_decimal", Schema.ColumnType.DECIMAL));
-    columns.add(new Schema.Column("type_date", Schema.ColumnType.DATE));
-    columns.add(new Schema.Column("type_timestamp", Schema.ColumnType.TIMESTAMP));
-
-    Schema schema = new Schema(columns);
-    Tuple2<SparkDDF, SchemaHandler.ApplySchemaStatistic> applySchema = schemaHandler.applySchema(schema);
+    List<Schema.Column> applyColumns = new ArrayList<>();
+    applyColumns.add(new Schema.Column("type_string", Schema.ColumnType.STRING));
+    applyColumns.add(new Schema.Column("type_boolean", Schema.ColumnType.BOOLEAN));
+    applyColumns.add(new Schema.Column("type_short", Schema.ColumnType.SMALLINT));
+    applyColumns.add(new Schema.Column("type_long", Schema.ColumnType.BIGINT));
+    applyColumns.add(new Schema.Column("type_double", Schema.ColumnType.DOUBLE));
+    applyColumns.add(new Schema.Column("type_decimal", Schema.ColumnType.DECIMAL));
+    applyColumns.add(new Schema.Column("type_date", Schema.ColumnType.DATE));
+    applyColumns.add(new Schema.Column("type_timestamp", Schema.ColumnType.TIMESTAMP));
+
+    Schema applySchema = new Schema(applyColumns);
+
+    testApplyWithDrop(ddf,applySchema);
+    testApplyWithoutDrop(ddf,applySchema);
+
+
+
+  }
+
+  private void testApplyWithoutDrop(SparkDDF ddf, Schema schema) throws DDFException {
+    SchemaHandler schemaHandler = (SchemaHandler) ddf.getSchemaHandler();
+    Tuple2<SparkDDF, SchemaHandler.ApplySchemaStatistic> applySchema = schemaHandler.applySchema(schema,false);
+    SparkDDF sparkDDF = applySchema._1();
+    SchemaHandler.ApplySchemaStatistic statistic = applySchema._2();
+    statistic.getMapColumnStatistic().forEach( (colName,colStats) -> {
+      System.out.println("column: " + colName);
```

Comment:
Should use logging instead of printing. Also, should we convert these println statements to assert ones? :)